src/shared/common/forms/composite/entity-list.jsx

```diff
@@ -96,6 +97,7 @@ class EntityList extends Component {
                             })}
                             <br />
                             {hasAddLink && <Link to={`/case/${page.params.caseId}/stage/${page.params.stageId}/entity/${entity}/add?hideSidebar=${hideSidebar}`} className="govuk-body govuk-link">Add a {entity}</Link>}
+                            {hasAddLinkNoMp && <Link to={`/case/${page.params.caseId}/stage/${page.params.stageId}/entity/${entity}/addNoMp?hideSidebar=${hideSidebar}`} className="govuk-body govuk-link">Add a {entity}</Link>}
```

Comment:
Can both of these ever be shown or is it one or the other?

As far as I can see here the difference in URL is the last bit at the end, so if only one is shown you could set that value as a variable with some logic and then only be having the Link component here once avoiding the duplication.